build/test.ps1

```diff
@@ -24,5 +24,8 @@ if ($Env:ENABLE_PYTHON -eq "false") {
   foreach ($PackageName in $PackageNames) {
     $EnvName = GetEnvName -PackageName $PackageName -CondaEnvironmentSuffix $CondaEnvironmentSuffix
     Invoke-Tests -PackageName $PackageName -EnvName $EnvName
```

Comment:
This will immediately stop if the tests of a package fail, without testing the rest of the packages.
It's typically better to actually run the tests of all packages so we can catch all failures in the same run.
Also, I'm not certain about the Powershell's semantics and if `$LastExitCode` will report the result of `Invoke-Tests` or the result of the last process (in this case `pytest`). My recommendation would be to change `Invoke-Test` to return $True on success and use that to check for package tests results.

Reply:
Thanks, that sounds good.
One issue I ran into is that if I have `Invoke-Tests` return a boolean, the output of `pytest` no longer shows up in the stdout, and if I pipe it to `Write-Host` instead I can no longer catch the `$LastErrorCode`. Do you have any recommendations on how to fix this issue?